### PR TITLE
refactor(argocdapp/devlake/trello/gitlabci/githubactions-nodejs): keep consistent with the `plugin template`

### PR DESCRIPTION
**Pull request by warren830**
## Tasks

> * [x]  argocdapp
> * [x]  devlake
> * [x]  trello
> * [x]  gitlabci
> * [x]  githubactions-nodejs

# Summary

1. argocdapp: rename validateOptions to validate, move to a standalone file validate.go
2. trello: finished
3. gitlabci: finished
4. githubactions: finished(extract workflow to upper layer)

# Related Issue

 #342

### CODE REVIEW

**Comment by daniel-hutao** [comment]
Related proposal: #342

**Review by daniel-hutao — COMMENTED** [review]
internal/pkg/plugin/githubactions/common.go → githubactions.go

no need to do it

**Review by IronCore864 — APPROVED** [review]

**Review by daniel-hutao — CHANGES_REQUESTED** [review]
Do not refactor the plugin logic at the same time as the `template refactoring`, please complete other refactorings in a separate pr, including the logic of `githubactions/xxx.go` can be moved to `internal/pkg/plugin/common/githubactions/ `.

File: internal/pkg/plugin/trello/read.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func Read(options map[string]interface{}) (map[string]interface{}, error) {
+
```
Comment (daniel-hutao) [inline internal/pkg/plugin/trello/read.go:12]: There is an unnecessary blank line here; but it doesn't matter, remember to modify it in the next pr

File: internal/pkg/plugin/trello/trello.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ type TrelloItemId struct {
 	doneListId  string
 }
 
-// CreateTrelloItems create board, and lists will be created automatically
+// CreateTrelloBoard CreateTrelloItems create board, and lists will be created automatically
```
Comment (daniel-hutao) [inline internal/pkg/plugin/trello/trello.go:24]: ？CreateTrelloBoard CreateTrelloItems？

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**